bin/sozo/src/commands/migrate.rs

```diff
@@ -99,6 +102,10 @@ impl MigrateArgs {
                 migration::migrate(&ws, world_address, rpc_url, account, &name, false, txn_config)
                     .await
             }),
+            MigrateCommand::GenerateOverlays => {
+                trace!(name, "Generating Overlay files.");
+                migration::generate_overlays(&ws)
+            }
```

Comment:
Extract this match as we've discussed as now this command requires katana to run.